From 62e272df848d6848482ceb534d14bf36c9fde241 Mon Sep 17 00:00:00 2001
From: Weng Xuetian <wengxt@gmail.com>
Date: Sat, 9 Jan 2021 19:07:24 -0800
Subject: [PATCH] Remove debug message to avoid polute to stdout.

This would break application like zenity which output to stdout.
---
 window-decorations-reload-module/reloader.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/window-decorations-reload-module/reloader.c b/window-decorations-reload-module/reloader.c
index 9e367dc..606bb1f 100644
--- a/window-decorations-reload-module/reloader.c
+++ b/window-decorations-reload-module/reloader.c
@@ -55,7 +55,6 @@ void manage_css_provider(GFileMonitor *monitor, GFile *file, GFile *other_file,
 
 void reload_css_provider()
 {
-    printf("WINDOW DECORATIONS RELOADED\n");
     if (css_provider != NULL) {
         remove_css_provider();
     }
-- 
GitLab
